operator/stringutil/strings.go

```diff
@@ -0,0 +1,21 @@
+package stringutil
+
+// Contains returns whether the string `s` is contained in `slice`.
+func Contains(slice []string, s string) bool {
+	for _, item := range slice {
+		if item == s {
+			return true
+		}
+	}
+	return false
+}
+
+// Remove returns a new slice with `s` removed from `slice`.
```

Comment:
This should document that it removes the _first_ instance, not all.

Reply:
Fixed with f83f9b5.